app/views/Cell.coffee

```diff
@@ -3,12 +3,32 @@ Colors = require './Colors'
 
 class Cell
 
-  constructor: (@col, @row, @size, @two, @board, @clickHandler) ->
+  constructor: (@col, @row, @size, @two, @board, @clickHandler, symbols, value) ->
```

Comment:
Pass in the specific SVG instead of all the symbols and the cell's value. (You might still need to pass in `value` for something)